lib/sharing/mutable_gist.dart

```diff
@@ -82,7 +85,18 @@ class MutableGist implements PropertyOwner {
     }
     return gist;
   }
-
+  
+  //Creates a Gist with summary element for sharing
+  Gist createGistWithSummary(String summary) {
```

Comment:
Let's remove this method, and instead add an optional named parameter to the method above (`createGist`), That would look like `{String summary}`.